flight/Modules/Actuator/actuator.c

```diff
@@ -251,6 +282,12 @@ int32_t ActuatorInitialize()
 	}
 #endif
 
+#if defined(PIOS_INCLUDE_ESCTELEMETRY)
+	if (ActuatorTelemetryInitialize() == -1) {
```

Comment:
Safe to go `< 0` or `!= 0` IMO, *Initialize might return multiple error codes at some point. :wink: Actually, [it already can return `-2`](https://github.com/d-ronin/dRonin/blob/next/flight/UAVObjects/uavobjecttemplate.c#L57), so this code is wrong.

Reply:
Value -2 means already initialized, which isn't a reason we'd like to fail for. The all other UAVO inits do the == -1 thing, too.